kv/txn_coord_sender.go

```diff
@@ -35,8 +35,11 @@ import (
 	"github.com/cockroachdb/cockroach/util/hlc"
 	"github.com/cockroachdb/cockroach/util/log"
 	gogoproto "github.com/gogo/protobuf/proto"
+	"github.com/montanaflynn/stats"
```

Comment:
if we needed an excuse for using it, we could probably do it here, but I remember that we weren't even sure whether it was our best bet and I'm not too inclined to get sidetracked.